docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3.9'
+services:
+  postgres:
+    container_name: op_ent_db
+    image: postgres:alpine
+    volumes:
+      - op_ent_db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: 'op_ent'

```

Comment:
Check if can be set using `.env` instead (as well as user and password)

Reply:
I don't think.

Reply:
Have a look at https://stackoverflow.com/questions/46753336/docker-compose-and-postgres-official-image-environment-variables and check if it works please

Reply:
It doesn't work for me.

Reply:
Okay, we can use `POSTGRES_USER=${POSTGRES_USER}` instead to load them from the .env